community-operators/microcks/microcks-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,206 @@
+      mediatype: image/png 
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: false
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+      - serviceAccountName: microcks-ansible-operator
+        rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - '*'
+        - apiGroups:
+          - extensions
+          resources:
+          - ingresses
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - microcks.github.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+      deployments:
+      - name: microcks-ansible-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: microcks-ansible-operator
+          template:
+            metadata:
+              labels:
+                name: microcks-ansible-operator
+            spec:
+              serviceAccountName: microcks-ansible-operator
+              containers:
+                - name: microcks-ansible-operator
+                  image: quay.io/microcks/microcks-ansible-operator
```

Comment:
Please use a tagged image

Reply:
Added in commit caddfc74bd5023fc82628e70d9ca8bd08a643bc8